go/flags/endtoend/vtctldclient.txt

```diff
@@ -30,6 +30,7 @@ Available Commands:
   ExecuteFetchAsApp           Executes the given query as the App user on the remote tablet.
   ExecuteFetchAsDBA           Executes the given query as the DBA user on the remote tablet.
   ExecuteHook                 Runs the specified hook on the given tablet.
+  ExecuteMultiFetchAsDBA      Executes given multiple queries as the DBA user on the remote tablet.
```

Comment:
For symmetry with the other `ExecuteFetch`, does this mean we also want / need `ExecuteMultiFetchAsApp`?

Reply:
@shlomi-noach My thinking is that it would be mostly for people using `ExecuteFetchAsApp` with multiple queries and have it kinda accidentally mostly work. They wouldn't have anywhere to go if we don't have it. I don't know if this is a real valid concern though or not.

Reply:
Having consulted with the team, we should not add `ExecuteMultiFetchAsApp` right now. Only when there's a specific use case.